src/internal/Modal.spec.js

```diff
@@ -51,6 +51,99 @@ describe('<Modal />', () => {
       assert.strictEqual(modal.is('div'), true, 'should be a div');
       assert.strictEqual(modal.hasClass(classes.modal), true, 'should have the modal class');
     });
+
+    describe('handlers', () => {
+      let instance;
+
+      before(() => {
+        instance = wrapper.instance();
+      });
+
+      describe('focus', () => {
+        before(() => {
+          instance.modal = spy();
+          instance.modal.lastChild = spy();
+          instance.modal.lastChild.setAttribute = spy();
+          instance.modal.lastChild.focus = spy();
+        });
+
+        describe('modalContent has tabIndex attribute', () => {
+          let consoleError;
+          let consoleErrorSpy;
+
+          before(() => {
+            instance.modal.lastChild.hasAttribute = stub().returns(true);
+            consoleError = console.error; // eslint-disable-line no-console
+            consoleErrorSpy = spy();
+            console.error = consoleErrorSpy; // eslint-disable-line no-console
```

Comment:
My motivation is around limitating the risk we take by monkey patching the console module. We don't want to miss any call.
I don't have any clear implementation in mind. After all, I'm fine with your version. But I'm not convinced that it's the best solution available.